apps/web/src/styles/typography.css

```
@@ -0,0 +1,83 @@
+@utility title-1 {
```

Stylelint (17.11.0)

[error] Unexpected unknown at-rule "@utility" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)